packages/core/tests/unit/models/report-test.js

```diff
@@ -145,6 +145,15 @@ module('Unit | Model | report', function (hooks) {
         unset(serialized, `data.attributes.request.columns[${idx}].cid`);
       });
 
+      // check and remove cid (b/c they're non-deterministic)
+      const sortCids = [];
+      serialized.data.attributes.request.sorts.forEach((sort, idx) => {
+        sortCids.push(sort.cid);
+        unset(serialized, `data.attributes.request.sorts[${idx}].cid`);
+      });
+      assert.equal(sortCids.length, 1, 'there is one sort');
+      assert.deepEqual(sortCids[0], cids[3], 'sort has same cid as matching column');
```

Comment:
We need to add a serializer for sort because this should not show up in the serialized request (or update the persistence ws to support it but I think it's ok to keep client side only for now)

```js
import BaseJsonSerializer from 'navi-core/serializers/base-json-serializer';

export default class SortSerializer extends BaseJsonSerializer {
  attrs: {
    cid: { serialize: false },
  }
}
```